views/peeps/index.erb

```diff
@@ -0,0 +1,9 @@
+<ul>
+  <% @peeps.each do |peep| %>
+    <li><%= peep %></li>
+  <% end %>
+</ul>
+
+<form method="get" action="/">
+    <button type="submit">Return to Homepage</button>
```

Comment:
Like the use of button to return to the home page rather than a link which is something I didn't use